test/code/providers/os_provider/osprovider_test.cpp

```diff
@@ -115,6 +115,17 @@ class OSProvider_Test : public CPPUNIT_NS::TestFixture
 
     void ValidateInstance(const TestableContext &context, std::wstring errMsg)
     {
+        bool fRunningOnTravis = false;
+#if defined(linux)
+        // We make assumptions about our environment, but those don't always hold on Travis
+
+        char *envTravis = getenv("TRAVIS");
```

Comment:
It would be cleaner to have the configure step check the environment for TRAVIS and define a `travis` macro symbol.
This would allow all parts of the code to use a simple ifdef.